pulsemeeter/__main__.py

```diff
@@ -460,12 +478,10 @@ def main():
         except Exception as ex:
             print('unable to close server', ex)
             return 1
-
-
+    
     else:
         create_parser_args()
         return 0
 
 
-    start_app(isserver, trayonly, server)
```

Comment:
I don't understand why remove it and call it from two different places (only no args and 'daemon' arg reach this part of the code anyway, the other commands will have already returned)